functions/export-csv.js

```diff
@@ -133,7 +133,9 @@ function getMultipleChoiceValues(id, element) {
  * Returns the file name in lowercase (replacing any special characters with '-') for csv export
  */
 function getFileName(layerName) {
-  return layerName.replace(/[^a-z0-9]/gi, "-").toLowerCase() + ".csv";
+  layerName =
```

Comment:
This will fail with an error if `layerName` is null or undefined. Please see previous suggestion to fix :)